dev/cnf/oss_dependencies.maven

```diff
@@ -349,6 +349,8 @@ org.apache.neethi:neethi:3.0.2
 org.apache.servicemix.bundles:org.apache.servicemix.bundles.bcel:5.2_3
 org.apache.thrift:libthrift:0.11.0
 org.apache.tomcat.embed:tomcat-embed-websocket:8.5.23
+org.apache.tomcat:tomcat-el-api:10.0.0-M7
```

Comment:
Shouldn't we be pulling in the Jakarta API?
https://repo1.maven.org/maven2/jakarta/el/jakarta.el-api/4.0.0-RC2/

Reply:
Although both use the same EL 4.0 spec, there are a few differences in the API code. 

The major issue is when testing the jakarta.el API with the Tomcat Jasper EL Impl under java 2 security, we receive a number of security warnings. I believe those should be addressed in the API code? The security manger does not throw the same warnings under the Tomcat API.  I opened an issue in the Eclipse EL Community for further feedback: https://github.com/eclipse-ee4j/el-ri/issues/131

Reply:
There is also an NPE thrown in app used for [testJSP23ResolutionVariableProperties](https://github.com/OpenLiberty/open-liberty/blob/9974f20f4c8930c981071b7d01dd1581b537db14/dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPTests.java#L715).

Otherwise,  I think it's best to use the Tomcat API for now and possible which later on if preferred.